apps/chatbot/src/modules/vector_database.py

```diff
@@ -41,16 +42,23 @@
 PROVIDER = os.getenv("CHB_PROVIDER")
 assert PROVIDER in ["google", "aws"]
 
+TODAY = date.today().strftime("%Y-%m-%d")
+INDEX_ID = get_ssm_parameter(os.getenv("CHB_LLAMAINDEX_INDEX_ID"))
+NEW_INDEX_ID = f"index--{TODAY}"
+
 REDIS_URL = os.getenv("CHB_REDIS_URL")
 WEBSITE_URL = os.getenv("CHB_WEBSITE_URL")
 REDIS_CLIENT = Redis.from_url(REDIS_URL, socket_timeout=10)
 REDIS_ASYNC_CLIENT = aredis.Redis.from_pool(
     aredis.ConnectionPool.from_url(REDIS_URL)
 )
 REDIS_INDEX_NAME = os.getenv("CHB_REDIS_INDEX_NAME")
-INDEX_ID = get_ssm_parameter(os.getenv("CHB_LLAMAINDEX_INDEX_ID"))
+EMBEDDING_DIMS = {
+    "models/text-embedding-004": 768,
+    "cohere.embed-multilingual-v3": 1024
+}
 REDIS_SCHEMA = IndexSchema.from_dict({
-    "index": {"name": REDIS_INDEX_NAME, "prefix": "index/vector"},
+    "index": {"name": f"{REDIS_INDEX_NAME}_{INDEX_ID}", "prefix": f"index_{INDEX_ID}/vector"},
     "fields": [
         {"name": "id", "type": "tag", "attrs": {"sortable": False}},
         {"name": "doc_id", "type": "tag", "attrs": {"sortable": False}},
```

Comment:
Is this actually needed anymore?

Reply:
it is for the the current schema. otherwise it would not load. the plan is to change it after a new re-indexing